### DISCUSSION TITLE
Fedora libraries installation

### DISCUSSION THREAD

**Discussion by mSamiolo** [cpmech/russell#40] [General]
Good morning, I was starting to use your crates in my code however using a Fedora system I have encountered a problem due to lack of libraries, resolved thanks to the installation of  lapack-devel and openblas-devel using:

`sudo dnf install lapack-devel openblas-devel `

Do you think this can cause the code to break?

**Comment by cpmech** [comment]
I think it should be all right.

However, if you intend to use `russell_sparse` to solve sparse linear systems, you will need _mumps_ and _suitesparse_.

If you don't plan to use `russell_sparse`, all should work just fine because no other crate depends on `russell_sparse`.
